components/dashboard/package.json

```diff
@@ -14,39 +14,38 @@
     "xterm-addon-fit": "^0.5.0"
   },
   "devDependencies": {
-    "@tailwindcss/forms": "^0.2.1",
-    "@tailwindcss/postcss7-compat": "^2.0.2",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
+    "@craco/craco": "^6.1.1",
+    "@tailwindcss/forms": "^0.3.2",
+    "@tailwindcss/postcss7-compat": "^2.1.0",
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.1.10",
     "@types/classnames": "^2.2.11",
-    "@types/jest": "^26.0.15",
+    "@types/jest": "^26.0.22",
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-router": "^5.1.12",
+    "@types/react-dom": "^17.0.3",
+    "@types/react-router": "^5.1.13",
     "@types/react-router-dom": "^5.1.7",
-    "@typescript-eslint/eslint-plugin": "^4.15.2",
-    "@typescript-eslint/parser": "^4.15.2",
+    "@typescript-eslint/eslint-plugin": "^4.21.0",
+    "@typescript-eslint/parser": "^4.21.0",
     "autoprefixer": "^9.8.6",
-    "classnames": "^2.2.6",
-    "eslint": "^7.21.0",
+    "classnames": "^2.3.1",
+    "eslint": "^7.23.0",
+    "eslint-config-react-app": "^6.0.0",
     "postcss": "^7.0.35",
-    "react-scripts": "4.0.2",
+    "react-scripts": "^4.0.3",
     "tailwind-underline-utils": "^1.1.2",
-    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.2",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.0",
     "tailwindcss-filters": "^3.0.0",
-    "typescript": "~4.1.2",
-    "web-vitals": "^1.0.1"
+    "typescript": "^4.2.4",
```

Comment:
why bumping TS version? 
it would be best to update for all components at the same time, otherwise we end up with different TS levels in the editor and build errors might occur.

Reply:
fixed